src/kernels/gpu_reference_kernel/naive_conv_gcn.s

```diff
@@ -1,10 +1,10 @@
 	.text
```

Comment:
@atamazov I think we'd better remove this `.s` file since above `naive_conv.cpp` is simple enough for our compilers to compile proper code, at least for recent generation of new asics. Every time change to the hip source `naive_conv.cpp` also need to change this `.s` which I think increase maintain efforts. How do you think of this?

Reply:
Personally, I appreciate this change and updated `findkernelfile` function in latest commit, but not delete the asm file yet. Wait response from @atamazov.